cmd/upload.go

```diff
@@ -53,4 +54,5 @@ func init() {
 	uploadCmd.Flags().Float64("hydration", 0, "Set your hydration in percent")
 	uploadCmd.Flags().Float64P("bone", "b", 0, "Set your bone mass in percent")
 	uploadCmd.Flags().Float64P("muscle", "m", 0, "Set your muscle mass in percent")
+	uploadCmd.Flags().Int64P("ts", "t", -1, "Set the timestamp of the measurement")
```

Comment:
Change `ts` to `unix-timestamp` I like this better as long positional argument as it describes itself better